src/codec_aom.c

```diff
@@ -921,6 +924,11 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
                 quantizerUpdated = AVIF_TRUE;
             }
         }
+        if (quantizerUpdated) {
+            // Make sure that maxQuantizer >= quantizer >= minQuantizer.
```

Comment:
Nit: Change `maxQuantizer` and `minQuantizer` to `rc_max_quantizer` and `rc_min_quantizer`


Add the following comment:

> If quantizer conflicts with rc_min_quantizer and rc_max_quantizer, quantizer has precedence.